cmd/cspi-mgmt/app/handler.go

```diff
@@ -188,7 +197,13 @@ func (c *CStorPoolInstanceController) updateStatus(cspi *apis.CStorPoolInstance)
 			OpenebsV1alpha1().
 			CStorPoolInstances(cspi.Namespace).
 			Update(cspi)
-		return err
+		if err != nil {
+			c.recorder.Event(cspi,
```

Comment:
we don't need to record event here.. its handled in the caller

Reply:
removed